docs/en/04_Changelogs/4.9.0.md

```diff
@@ -2,13 +2,130 @@
+| cwp/watea-theme | 3.1.0 |
+| dnadesign/silverstripe-elemental | 4.7.0 |
+| dnadesign/silverstripe-elemental-userforms | 3.0.0 |
+| silverstripe/akismet | 4.1.0 |
+| silverstripe/auditor | 2.3.0 |
+| silverstripe/blog | 3.8.0 |
+| silverstripe/ckan-registry | 1.3.0 |
+| silverstripe/comment-notifications | 2.1.0 |
+| silverstripe/comments | 3.6.0 |
+| silverstripe/content-widget | 2.2.0 |
+| silverstripe/contentreview | 4.3.0 |
+| silverstripe/controllerpolicy | 2.2.0 |
+| silverstripe/crontask | 2.3.0 |
+| silverstripe/documentconverter | 2.1.1 |
+| silverstripe/elemental-bannerblock | 2.3.0 |
+| silverstripe/elemental-fileblock | 2.2.0 |
+| silverstripe/environmentcheck | 2.3.0 |
+| silverstripe/externallinks | 2.1.1 |
+| silverstripe/fulltextsearch | 3.8.0 |
+| silverstripe/gridfieldqueuedexport | 2.5.0 |
+| silverstripe/html5 | 2.1.0 |
+| silverstripe/hybridsessions | 2.3.0 |
+| silverstripe/iframe | 2.1.0 |
+| silverstripe/ldap | 1.2.1 |
+| silverstripe/mfa | 4.4.0 |
+| silverstripe/realme | 4.1.1 |
+| silverstripe/registry | 2.3.0 |
+| silverstripe/restfulserver | 2.3.0 |
+| silverstripe/security-extensions | 4.1.0 |
+| silverstripe/securityreport | 2.3.0 |
+| silverstripe/segment-field | 2.4.0 |
+| silverstripe/sharedraftcontent | 2.5.0 |
+| silverstripe/sitewidecontent-report | 3.1.0 |
+| silverstripe/spamprotection | 3.1.0 |
+| silverstripe/spellcheck | 2.2.1 |
+| silverstripe/subsites | 2.4.0 |
+| silverstripe/tagfield | 2.7.0 |
+| silverstripe/taxonomy | 2.2.0 |
+| silverstripe/textextraction | 3.2.0 |
+| silverstripe/totp-authenticator | 4.2.0 |
+| silverstripe/userforms | 5.10.0 |
+| silverstripe/versionfeed | 2.1.0 |
+| silverstripe/webauthn-authenticator | 4.3.0 |
+| silverstripe/widgets | 2.1.1 |
+| symbiote/silverstripe-advancedworkflow | 5.5.0 |
+| symbiote/silverstripe-multivaluefield | 5.1.0 |
+| symbiote/silverstripe-queuedjobs | 4.8.0 |
+| tractorcow/silverstripe-fluent | 4.5.1 |
+
+</details>
+
+Upgrading to Recipe {{ version }} is recommended for all sites. This upgrade can be carried out by any development team familiar with Silverstripe.
+
+- [Security audit and regression test](#audit)
+- [For development teams of Common Web Platform projects](#cwp-end)
+- [Dropping support for Internet Explorer 11](#ie11eol)
 - [Features and enhancements](#features-and-enhancements)
     - [Image lazy loading](#image-lazy-loading)
     - [Manage your CMS sessions across devices](#session-manager)
     - [Default mail transport upgraded to sendmail](#sendmail)
+    - [Support for silverstripe/graphql v4](#graphqlv4)
     - [Other new features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Security audit and regression test{#audit}
+
+This release has gone through a regression test suite to identify any possible instability between modules.
+
+The release has also been audited by an independant security firm to identify possible security regressions.
+
+Review the [security audit report](INSERT-LINK-TO-AUDIT).
```

Comment:
```suggestion
## Regression test and Security audit{#audit}

This release has been comprehensively regression tested and passed to a third party for a security-focused audit. 

While it is still advised that you perform your own due diligence when upgrading your project, this work is performed to ensure a safe and secure upgrade with each recipe release.
```

Reply:
I agree that we should look to provide the audit report however we're not in a position to do this now. Based on my experience, the report and vendor response can require follow-up conversations which will need some further thinking on how to support that in a channel like this.

In lieu of a report, I've tried to tweak the wording to explain the benefit of this in another way.